lib/util/fs/util.ts

```diff
@@ -3,31 +3,39 @@ import { GlobalConfig } from '../../config/global';
 import { FILE_ACCESS_VIOLATION_ERROR } from '../../constants/error-messages';
 import { logger } from '../../logger';
 
-function assertBaseDir(path: string, baseDir: string): void {
+function assertBaseDir(path: string, baseDir: string, debug = false): void {
   if (!path.startsWith(upath.resolve(baseDir))) {
     logger.warn(
       { path, baseDir },
       'Preventing access to file outside the base directory'
     );
     throw new Error(FILE_ACCESS_VIOLATION_ERROR);
   }
+  // istanbul ignore if
+  if (debug && (path.endsWith('passwd') || path.endsWith('bar'))) {
+    throw { path, baseDir };
+  }
```

Comment:
i don't like this. Why do we need such strange thing?

Reply:
Because I'm trying to figure out what's going on in the Windows platform to which I have no any access other than CI